README.md

```diff
@@ -176,95 +136,12 @@ We are looking for maintainers to add more parsers and to write query files for
 
 For related information on the supported languages, including related plugins, see [this wiki page](https://github.com/nvim-treesitter/nvim-treesitter/wiki/Supported-Languages-Information).
 
-# Available modules
```

Comment:
My plan was to keep that information in the form of "to enable <foo>, put this in your config".

But I can just re-add that later.